src/pages/workspaces/List.js

```diff
@@ -233,15 +230,8 @@ export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
         })
       ]),
       h(PageFadeBox, [
-        div({
-          style: {
-            display: 'flex', alignItems: 'flex-end', justifyContent: 'space-between',
-            marginBottom: '1rem'
-          }
-        }, [
-          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, [
-            'Workspaces'
-          ]),
+        div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between', marginBottom: '1rem' } }, [
+          div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, ['Workspaces']),
```

Comment:
All 3 of these are upper case, which makes me feel like they should be `pageTitle` styled instead of `sectionHeader`. But, the designs do say size 16 / weight 600.